### PR TITLE
Swap 1626 click on scheduled event

### PR DESCRIPTION
**Pull request by martin-trajanovski**
## Description

When scheduled event clicked on the calendar view it should open the same view that you see when event is booked and being able to edit the event when you click on it.

## How Has This Been Tested

e2e tests
manual tests

## Fixes

https://jira.esss.lu.se/browse/SWAP-1626

## Tests included/Docs Updated?

- [x] I have added tests to cover my changes.
- [ ] All relevant doc has been updated

### CODE REVIEW

File: src/components/equipment/CreateEditEquipment.tsx [hunk 1]
```diff
@@ -196,7 +193,7 @@ export default function CreateEditEquipment() {
                   });
 
                   error
-                    ? enqueueError(error)
+                    ? enqueueError(error as ResourceId)
```
Comment (jekabs-karklins) [inline src/components/equipment/CreateEditEquipment.tsx:196]: The `error` is an arbitrary string containing the description of the error, not a ResourceId anymore

File: src/components/equipment/CreateEditEquipment.tsx [hunk 2]
```diff
@@ -206,10 +203,10 @@ export default function CreateEditEquipment() {
...
```
Comment (jekabs-karklins) [inline src/components/equipment/CreateEditEquipment.tsx:206]: Same here

**Review by jekabs-karklins — CHANGES_REQUESTED** [review]

File: src/components/equipment/CreateEditEquipment.tsx (hunk 1 above)
Comment (martin-trajanovski) [reply src/components/equipment/CreateEditEquipment.tsx:196]: Not sure how this works if it is implemented here on the scheduler

Comment (martin-trajanovski) [reply src/components/equipment/CreateEditEquipment.tsx:196]: I still see that on the scheduler-backend we have the old ResourceId as a reason returned

Comment (martin-trajanovski) [reply src/components/equipment/CreateEditEquipment.tsx:196]: Maybe there should be a separate task for this

**Pull request merged**